src/features/groups/components/InviteMemberDialog/InviteMemberDialogContent.tsx

```diff
@@ -0,0 +1,46 @@
+import { LinkIcon } from '@/assets/images/icons';
+import {
+  Button,
+  DialogBody,
+  DialogContent,
+  DialogHeader,
+  DialogTitle,
+  Icon,
+} from '@/components/ui';
+
+import { useEffect, useState } from 'react';
+
+export const InviteMemberDialogContent = ({ isDialogOpen }: { isDialogOpen: boolean }) => {
+  const [isCopyDisabled, setIsCopyDisabled] = useState(false);
+  const handleLinkCopy = () => {
```

Comment:
Please add TSDoc to this function, although it's very simple.

TSDoc is always required whenever you create a function or interface, as per our current coding rules. I am sorry, but it should have been documented somewhere...

Reply:
@nick-y-ito 
I added TSDoc to the function , please take a look!